api/src/main/java/com/datatorrent/api/StreamingApplication.java

```diff
@@ -47,6 +47,12 @@
   String ENVIRONMENT = DT_PREFIX + "environment";
 
   /**
+   * Use this config variable to to add any external library to classpath for the DAG.
+   * Value of this variable is same as StramAppLauncher.LIBJARS_CONF_KEY_NAME.
+   */
+  String LIBJARS_CONF_KEY_NAME = "tmplibjars";
```

Comment:
There already is an attribute that was taken out of the public API some time ago... LogicalPlan.LIBRARY_JARS